### PR TITLE
feat(Logs): Added Vercel integration

### PR DESCRIPTION
**Pull request by nbaenam**
Created a new page for the Vercel integration.

This is the [ticket (NR-88475)](https://issues.newrelic.com/browse/NR-88475)

### CODE REVIEW

File: src/content/docs/logs/forward-logs/vercel-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+import logsVercelAllowPermissions from 'images/logs_screenshot-crop_vercel-allow-permissions.png'
+
+import logsVercelSetLicenseKey from 'images/logs_screenshot-crop_vercel-set-license-key.png'
+
+import logsVercelGetTraceData from 'images/logs_screenshot-crop_vercel-get-trace-data.png'
+
+
+
+Use New Relic to analize and storage the ingested logs from your Vercel projects and traces from your serverless functions.
```
Comment (stumblefiend) [inline src/content/docs/logs/forward-logs/vercel-integration.mdx:25]:
```suggestion
Use New Relic to analyze and store the ingested logs from your Vercel projects and traces from your serverless functions.
```

File: src/content/docs/logs/forward-logs/vercel-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+Install the quickstart to get a pre-built dashboard and alert to monitor the performance of your functions. You'll also get a pre-built alert that triggers when the error rate (4xx/5xx errors) exceeds 1%. Follow these instructions:
+
+1. Go to the [Vercel quickstart](https://newrelic.com/instant-observability/vercel) in New Relic Instant Observability and click **Install now**.
+2. Select an account and click **Begin installation**.
+3. If you've already completed the [Vercel configuration steps](#configuration), select **Done** to move onto the next step.
+4. The quickstart deploys the resources to your account. Click **See your data** to get to the dashboard.
+
+The Vercel quickstart dashboard visualizes all your key data into three pages. Click the **Overview**, **Build Logs**, and **Serverless Function** tabs to get an overview, analyze build logs, and dive deep into your Serverless Function performance.
+
+**Add here the dashboard screenshot**
```
Comment (stumblefiend) [inline src/content/docs/logs/forward-logs/vercel-integration.mdx:125]: ???

File: src/content/docs/logs/forward-logs/vercel-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+## Optional: Get trace data from Vercel with OpenTelemetry [#vercel-opentelemetry]
+
+You can use [OpenTelemetry](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/) to import trace data from your applications running in Vercel functions. To set this option, follow these steps:
+
+1. After clicking **Add integration** in the [step 5](#step5) in the [Vercel configuration](#configuration), you'll see the configuration details dialog box.
```
Comment (stumblefiend) [inline src/content/docs/logs/forward-logs/vercel-integration.mdx:131]:
```suggestion
1. After clicking **Add integration** in [step 5](#step5) in the [Vercel configuration](#configuration), you'll see the configuration details dialog box.
```

File: src/content/docs/logs/forward-logs/vercel-integration.mdx (hunk 2 above)
Comment (nbaenam) [reply src/content/docs/logs/forward-logs/vercel-integration.mdx]: Yes, it is. I'm waiting for an environment and take it.

File: src/content/docs/logs/forward-logs/vercel-integration.mdx (hunk 3 above)
Comment (nbaenam) [reply src/content/docs/logs/forward-logs/vercel-integration.mdx:131]: Done it. Thanks!

**Review by louisleungk — APPROVED** [review]
Minor modification to caption. Looking great, thank you!

File: src/content/docs/logs/forward-logs/vercel-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+<img
+    width="80%;"
+    title="Vercel dashboard"
+    alt="Vercel dashboard"
+    src={logsVercelDashboard}
+  />
+
+<figcaption>
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Dashboards > Vercel** to see your Vercel dashboard.
```
[the remark this answers is not in the document]
Comment (nbaenam) [reply src/content/docs/logs/forward-logs/vercel-integration.mdx:38]: Good catch!

**Review by homelessbirds — APPROVED** [review]

**Comment by homelessbirds** [comment]
Looks good :)

**Pull request merged**